platforms/minestom/src/main/java/com/dfsek/terra/minestom/chunk/GeneratedChunkCache.java

```diff
@@ -0,0 +1,55 @@
+package com.dfsek.terra.minestom.chunk;
+
+import com.dfsek.terra.api.util.generic.pair.Pair;
+
+import com.dfsek.terra.api.world.ServerWorld;
+import com.dfsek.terra.api.world.biome.generation.BiomeProvider;
+import com.dfsek.terra.api.world.chunk.generation.ChunkGenerator;
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+import com.github.benmanes.caffeine.cache.LoadingCache;
+import com.github.benmanes.caffeine.cache.stats.CacheStats;
+import net.minestom.server.world.DimensionType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class GeneratedChunkCache {
```

Comment:
Why is this needed?

Reply:
Minestom does not implement separate stages between terrain and feature generation as of now. This is needed to store terrain for "ungenerated" chunks for feature generation, since we can't just load them via minestom